src/systems/joint_controller/JointController.cc

```diff
@@ -98,6 +98,13 @@ void JointController::Configure(const Entity &_entity,
     return;
   }
 
+  gazebo::Entity jointEntity = this->dataPtr->model.JointByName(_ecm, this->dataPtr->jointName);
+  if (jointEntity == kNullEntity)
+  {
+    ignerr << "Joint with name[" << this->dataPtr->jointName << "] not found. "
+    << "The JointController will not control this joint.\n";
+  }
```

Comment:
if this an error we should probably return here

Reply:
@ahcorde I'm open to changing it to an error. Could you explain how to do thaat?

I based this on https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo5/src/systems/joint_state_publisher/JointStatePublisher.cc#L75 which doesn't display an error but just prints to cerr as well